README.md

```diff
@@ -7,17 +7,36 @@ govet examines Go source code and reports suspicious constructs, such as Printf
 ### Installation
 
 1. If you haven't already, [install the Code Climate CLI](https://github.com/codeclimate/codeclimate).
-2. Run `codeclimate engines:enable govet`. This command both installs the engine and enables it in your `.codeclimate.yml` file.
-3. You're ready to analyze! Browse into your project's folder and run `codeclimate analyze`.
+2. Add the following to your Code Climate config:
+  ```yaml
+  plugins:
+    govet:
+      enabled: true
+  ```
+3. Run `codeclimate engines:install`
+4. You're ready to analyze! Browse into your project's folder and run `codeclimate analyze`.
 
 ### Building
 
 ```console
-./bin/build
+make image
 ```
 
 This will build a `codeclimate/codeclimate-govet` image locally
 
+### Updating
+
+`govet` is a part of the Go distribution and shares version with it. Once in a
+while a new version of Go gets packaged and in order to get the latest version
+and force a new docker image build, we have to update the `engine.json` file.
+
+```console
+make update
```

Comment:
There's a presumption in these instructions that you have the latest `golang:alpine` image where you run `make update`, but I don't think that's reliable. If you had built an older version of this library on your laptop & then a new go version was released you'd probably still have the older `golang:alpine` on your laptop when you got around to this unless you'd explicitly pulled it. Is it worth adding a `docker pull golang:alpine` to these steps?